index.html

```diff
@@ -153,6 +153,22 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
           };
         </pre>
         <dl data-link-for="CropTarget" data-dfn-for="CropTarget"></dl>
+        <p>
+          To <dfn data-export>create a CropTarget</dfn> with <var>element</var> as input, run the following steps:
+          <ol>
+            <li>
+              <p>Let <var>cropTarget</var> be a new object of type {{CropTarget}}.</p>
+            </li>
+            <li>
+              <p>Let <var>weakRef</var> be a weak reference to <var>element</var>.</p>
```

Comment:
Can you suggest a link that we can use for the term "weak reference"?

Reply:
Those are two terms: "weak" and "reference".

The precedent here is [HTML](https://html.spec.whatwg.org/#offscreencanvas-placeholder) which uses the word "weak" 12 times without defining it, and uses the word "reference" 176 times.

This PR seems consistent with precedent. The term "weak" is used in [TAG design principles](https://w3ctag.github.io/design-principles/#js-gc) related to garbage collection, so it seems clear that a "reference" is implicitly strong (affects GC) unless it is preceeded by the word "weak". cc @annevk